shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/dml/MergeStatement.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.sql.parser.sql.common.statement.dml;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.generic.table.SimpleTableSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Merge statement.
+ */
+@Getter
+@Setter
+@ToString
+public abstract class MergeStatement extends AbstractSQLStatement implements DMLStatement {
+    
+    private final List<SimpleTableSegment> tables = new LinkedList<>();
```

Comment:
@tristaZero Is the column information needed to add in the MergeStatement?

Reply:
Hi @wgy8283335  I think so.

Generally, table segment, column segment (this fundamental info of a SQL) are necessary to put into `XXXStatement`, similar to `SelectStatement`. 
However, this is a new type of SQLStatement, which has no subsequent handling by ShardingSphere. Hence, I suggest we visit the primary segments (Like table or column) from its AST to the corresponding SQLStatement. 

@ThanoshanMV , Follow your pace. You can choose to define a basic `MergeStatement` to make it work well wholly and later fill more segments into it little by little.

Reply:
Okay @tristaZero, I'll do them.